0110    javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,14 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  const array = [];
+
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] != null) {
```

Comment:
Though it techincally doesn't make a difference here, it is better to get in the habit of using strict comparisons like `!==` that check value & type.